packages/shared/components/atoms/SideDrawer.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+    import { fade, fly } from 'svelte/transition'
+    import { appSettings } from '@core/app'
+
+    export let onClose: () => unknown = () => {}
```

Comment:
Might be good to create a type for these types of functions, e.g. `type EventFn = (..._args: unknown[]) => void` since we use them a lot. I think in most if not all cases they always are `void`.